.github/workflows/deploy.yml

```diff
@@ -85,15 +85,16 @@ jobs:
 
       - name: Terraform init
         run: |
-          terraform init -backend-config=terraform/workspace-variables/${{ env.DEPLOY_ENV }}_backend.tfvars terraform
+          cd terraform
```

Comment:
You could use `working-directory: terraform` instead. Here and in the other steps

Reply:
I could if you want, but i feel perhaps not necessary for single action? Also, it would make the target list longer? Just my thought :-)

Reply:
done...